source/administration/upgrade.rst

```diff
@@ -119,13 +119,12 @@ Location of your local storage directory
 
      sudo systemctl start mattermost
 
-#. Upgrade your ``config.json`` schema:
+#. If you are using an HA deployment, please apply the steps above on all the nodes in your cluster. After that, the "Config File MD5" column in the "High Availability" should be green. If they are yellow, please ensure that all nodes have the same server version and the same config. If they still show yellow, then there's a simple trick to trigger a config propagation across the cluster.

```

Comment:
```suggestion
#. If you're using a High Availability deployment you need to apply the steps above on all the nodes in your cluster. Once complete, the **Config File MD5** columns in the High Availability interface should be green. If they're yellow, please ensure that all nodes have the same server version and the same configuration. 

If they still show yellow, then you need to trigger a config propagation across the cluster:
```

Reply:
Ah my bad, it should be "High Availability section".